cash-register.js

```diff
@@ -6,102 +6,199 @@
 
```

Comment:
Yep, only put code that's relevant in git. Depends on the usefulness of the "copy" for yourself? If it's useful then keep it in git and keep it well documented (as it's basically a knowledge document at that point). If it's not providing use then get rid of it. :)